jadx-gui/src/main/java/jadx/gui/ui/tab/TabComponent.java

```diff
@@ -56,14 +59,15 @@ public void loadSettings() {
 	}
 
 	private Font getLabelFont() {
-		return tabbedPane.getMainWindow().getSettings().getFont().deriveFont(Font.BOLD);
+		return tabsController.getMainWindow().getSettings().getFont().deriveFont(Font.BOLD);
 	}
 
 	private void init() {
 		setLayout(new FlowLayout(FlowLayout.LEFT, 0, 0));
 		setOpaque(false);
 
-		JNode node = contentPanel.getNode();
+		JNode node = getNode();
+		TabBlueprint blueprint = getBlueprint();
```

Comment:
`blueprint` not used later in code.

Also, looks like we don't have any visual mark that tab was bookmarked. I think we can add a little bookmark icon in top left corner of tab icon. Please check `jadx.gui.utils.OverlayIcon` and how it used to add small icons for fields.